Src/Support/Google.Apis.Auth.AspNetCore3.IntegrationTests/Views/Home/ScopeListing.cshtml

```diff
@@ -9,23 +11,25 @@
 <html>
 <head>
     <meta name="viewport" content="width=device-width" />
-    <title>Integreation tests - Scope Listing</title>
+    <title>Google.Apis.Auth.AspNetCore3 sample - Scope Listing</title>
 </head>
 <body>
-    <h1>Integration tests - Scope Listing</h1>
-    <ul>
-        @foreach (var scope in @Model)
-        {
-        <li>@scope</li>
-        }
-    </ul>
+    <h2>Google.Apis.Auth.AspNetCore3 sample - Scope Listing</h2>
     @if (@Model.Count > 0)
     {
-    <p>Test passed</p>
+        <p>@User.FindFirst(ClaimTypes.GivenName)?.Value these are the scopes this application
```

Comment:
Possibly put the user name in as a separate paragraph instead of including it here? It looks a little odd to me - not a big deal though.

Reply:
OK, I'll add You are logged in as ... and then the specific paragraph on each section.